### PR TITLE
[Debug] Same vendor namespaces remapping

### PR DESCRIPTION
**Pull request by fancyweb**
Doc for https://github.com/symfony/symfony/pull/30899.

### CODE REVIEW

File: components/debug.rst [hunk 1]
```diff
@@ -88,4 +88,34 @@ Using the ``DebugClassLoader`` is done by calling its static
 
     DebugClassLoader::enable();
 
+The ``DebugClassLoader`` also does lightweight static code analysis on the
```
Comment (Simperfit) [inline components/debug.rst]: does seems not that needed

File: components/debug.rst [hunk 2]
```diff
@@ -88,4 +88,34 @@ Using the ``DebugClassLoader`` is done by calling its static
...
+classes it loads in order to generate some useful notice messages. For example,
```
Comment (Simperfit) [inline components/debug.rst]: it loads, in order to

File: components/debug.rst [hunk 3]
```diff
@@ -88,4 +88,34 @@ Using the ``DebugClassLoader`` is done by calling its static
...
+if one of your class extends a Symfony class that is marked with the
```
Comment (Simperfit) [inline components/debug.rst]: with `@internal`

**Review by Simperfit — CHANGES_REQUESTED** [review]

File: components/debug.rst (hunk 1 above)
Comment (fancyweb) [reply components/debug.rst]: What do you mean? I don't get it ^^

Comment (fancyweb) [reply components/debug.rst]: Thank you for reviewing btw!

Comment (Simperfit) [reply components/debug.rst]: the word "does" seems not that needed

Comment (fancyweb) [reply components/debug.rst]: What other verb would you use? If I remove "does", the sentence has no verb ^^

**Pull request closed without merging**